sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -63,7 +63,8 @@ export class AssessmentAESTController extends BaseController {
         ApplicationExceptionStatus.Pending,
         ApplicationExceptionStatus.Declined,
       );
-    if (applicationExceptions) {
+
+    if (applicationExceptions && applicationExceptions.length > 0) {
```

Comment:
Thanks for doing the fix. Spoiler alert this is the original code that causes PR comments and I end up changing it in the wrong way and introducing the issue 😉 
Please change as below, an empty result will be returned as an empty array.
```typescript
if (applicationExceptions.length > 0)
```